auto-generated/vector-crypto/intrinsic_funcs/01_zvbc_-_vector_carryless_multiplication.md

```diff
@@ -0,0 +1,185 @@
+
+## Zvbc - Vector Carryless Multiplication:
+
+### [Vector Carryless Multiplication]():
+
+**Prototypes:**
+``` C
+vuint8mf8_t __riscv_vclmul_vv_u8mf8 (vuint8mf8_t vs2, size_t vl);
+vuint8mf8_t __riscv_vclmul_vx_u8mf8 (vuint8mf8_t vs2, size_t vl);
+vuint8mf4_t __riscv_vclmul_vv_u8mf4 (vuint8mf4_t vs2, size_t vl);
+vuint8mf4_t __riscv_vclmul_vx_u8mf4 (vuint8mf4_t vs2, size_t vl);
+vuint8mf2_t __riscv_vclmul_vv_u8mf2 (vuint8mf2_t vs2, size_t vl);
+vuint8mf2_t __riscv_vclmul_vx_u8mf2 (vuint8mf2_t vs2, size_t vl);
+vuint8m1_t __riscv_vclmul_vv_u8m1 (vuint8m1_t vs2, size_t vl);
+vuint8m1_t __riscv_vclmul_vx_u8m1 (vuint8m1_t vs2, size_t vl);
+vuint8m2_t __riscv_vclmul_vv_u8m2 (vuint8m2_t vs2, size_t vl);
+vuint8m2_t __riscv_vclmul_vx_u8m2 (vuint8m2_t vs2, size_t vl);
+vuint8m4_t __riscv_vclmul_vv_u8m4 (vuint8m4_t vs2, size_t vl);
+vuint8m4_t __riscv_vclmul_vx_u8m4 (vuint8m4_t vs2, size_t vl);
+vuint8m8_t __riscv_vclmul_vv_u8m8 (vuint8m8_t vs2, size_t vl);
+vuint8m8_t __riscv_vclmul_vx_u8m8 (vuint8m8_t vs2, size_t vl);
+vuint16mf4_t __riscv_vclmul_vv_u16mf4 (vuint16mf4_t vs2, size_t vl);
+vuint16mf4_t __riscv_vclmul_vx_u16mf4 (vuint16mf4_t vs2, size_t vl);
+vuint16mf2_t __riscv_vclmul_vv_u16mf2 (vuint16mf2_t vs2, size_t vl);
+vuint16mf2_t __riscv_vclmul_vx_u16mf2 (vuint16mf2_t vs2, size_t vl);
+vuint16m1_t __riscv_vclmul_vv_u16m1 (vuint16m1_t vs2, size_t vl);
+vuint16m1_t __riscv_vclmul_vx_u16m1 (vuint16m1_t vs2, size_t vl);
+vuint16m2_t __riscv_vclmul_vv_u16m2 (vuint16m2_t vs2, size_t vl);
+vuint16m2_t __riscv_vclmul_vx_u16m2 (vuint16m2_t vs2, size_t vl);
+vuint16m4_t __riscv_vclmul_vv_u16m4 (vuint16m4_t vs2, size_t vl);
+vuint16m4_t __riscv_vclmul_vx_u16m4 (vuint16m4_t vs2, size_t vl);
+vuint16m8_t __riscv_vclmul_vv_u16m8 (vuint16m8_t vs2, size_t vl);
+vuint16m8_t __riscv_vclmul_vx_u16m8 (vuint16m8_t vs2, size_t vl);
```

Comment:
Why defining the `vcmul*` intrinsincs for anything else than `64-bit` elements ?